### PR TITLE
Unify binary path and config path choice

### PR DESCRIPTION
**Pull request by MaciejTe**
Unify binary path and config path choice

Changelog: None

Signed-off-by: Maciej Tomczuk <maciej.tomczuk@northern.tech>

### CODE REVIEW

**Comment by MaciejTe** [comment]
@mender-test-bot start pipeline

File: testutils/infra/cli.py [hunk 1]
```diff
@@ -39,27 +40,40 @@ def __init__(
...
+        """Choose binary and configuration paths depending on service flavour. """
+        for service in service_flavours:
+            try:
+                self.container_manager.execute(self.cid, [service.bin_path, "--help"])
+                self.service = service
+                break
+            except:
+                continue
+        else:
+            raise RuntimeError(f"no runnable binary found in mender-{service_name}")
```
Comment (lluiscampos) [inline testutils/infra/cli.py:55]: `service_name` already has the `mender-` prefix, so this should be:
```suggestion
            raise RuntimeError(f"no runnable binary found in {service_name}")
```

**Review by lluiscampos — APPROVED** [review]
Nice clean-up :+1:

File: testutils/infra/cli.py (hunk 1 above)
Comment (MaciejTe) [reply testutils/infra/cli.py]: done, thanks!

**Review by lluiscampos — APPROVED** [review]

**Pull request merged**